modules/gtksourceview/overlay.nix

```
@@ -0,0 +1,38 @@
+{ config, lib, ... }:
+
+let
+  inherit (lib) optionalAttrs;
+  inherit (config.lib.stylix) colors mkEnableTarget;
+
+  cfg = config.stylix.targets.gtksourceview;
+
+  style = colors {
+    template = ./template.xml.mustache;
+    extension = ".xml";
+  };
+
+  attrsOverride = version: oldAttrs: {
+    postFixup = ''
+      ${oldAttrs.postFixup or ""}
+      styles_dir="$out/share/gtksourceview-${version}/styles"
+      mkdir --parents "$styles_dir"
+      cp ${style} "$styles_dir/stylix.xml"
+    '';
+  };
+in
+{
+  options.stylix.targets.gtksourceview = {
+    enable = mkEnableTarget "GTKSourceView" true;
+  };
+
+  overlay =
+    _final: prev:
+    optionalAttrs (config.stylix.enable && cfg.enable) {
```

would ideal come from mkTarget
```suggestion
  overlay =
    _final: prev:
    optionalAttrs (config.stylix.enable && options.stylix.targets ? enable && cfg.enable) {
```

---

@brckd this needs to be added, the current code will cause an error

---

I hope that [3bfdcc1](https://github.com/nix-community/stylix/pull/958/commits/3bfdcc1c0b0ae92a10737b5bb010e64fff0f91b2) fixes it. Not sure how to test that though.